### PR TITLE
Fix accessibility issues due to aria-hidden and aria-labelledby in FileContent and CarouselFilmStrip

### LINKED ISSUE #3810: aria-hidden-focus: Ensures aria-hidden elements do not contain focusable elements
## Version
4.12.1, npm

## Describe the bug

- When navigating transcript, attachment uploads only reads “1 attachment” when on ‘activity focus’
- After pressing enter on the attachment, nothing is read aloud by AT
- Aria-hidden value is set on the anchor inside the attachment

## Steps to reproduce

1. Open MockBot
2. type 'file', 'document-word', 'document-plain', and/or 'document-data-uri'
3. Chrome/NVDA: 
   - When navigating transcript, attachment uploads only reads “1 attachment” when on ‘activity focus’
   - Enter to interact is not read
   - After pressing enter on the attachment, nothing is read aloud by AT
   - Also repros https://github.com/microsoft/BotFramework-WebChat/issues/3804 behavior
   - Note: `aria-live` behavior works as expected
4. Edge/Narrator:
   - 'Enter to interact' is read as expected
   - Pressing enter on the attachment says "Nothing to interact" or something similar
   - `aria-live` behavior seems as expected
   

## Expected behavior
When focus is on the attachment, "1 attachment, Press enter to interact" should be read.
When `aria-selected` is on the attachment, content of the anchor should be read

## Additional context

[Bug]

### LINKED ISSUE #3811: aria-valid-attribute: Ensures all ARIA attributes have valid values
Files as P0 due to deadline; as a need to fix it is P1
## Version
Related to https://github.com/microsoft/BotFramework-WebChat/pull/3098

## Describe the bug

`aria-labelledby` should be removed from hidden element

## Steps to reproduce

1. Open MockBot
2. Type/send 'carousel'
3. Use Fastpass and see that `aria-labelledby` is invalid

## Expected behavior

`aria-labelledby` should not be a present attribute

[Bug]

### PR DESCRIPTION
**Pull request by corinagum**
> Fixes #3810 
> Fixes #3811 

## Changelog Entry

-  Fixes [#3810](https://github.com/microsoft/BotFramework-WebChat/issues/3810). Removes `aria-hidden` from elements that have a focusable child, by [@corinagum](https://github.com/corinagum) in PR [#3836](https://github.com/microsoft/BotFramework-WebChat/pull/3836)
-  Fixes [#3811](https://github.com/microsoft/BotFramework-WebChat/issues/3811). Removes erroneous `aria-labelledby` from carousel, by [@corinagum](https://github.com/corinagum) in PR [#3836](https://github.com/microsoft/BotFramework-WebChat/pull/3836)

## Description

This contains a few accessibility fixes when running FastPass on Web Chat.

## Specific Changes

#3810 - Ensures aria-hidden elements do not contain focusable elements
- `FileContent.js`: Add `aria-label` `alt` text to download link, which was previously not being read.
- `FileContent.js`: Remove erroneous `aria-hidden=true`
- `FileContent.js`: Remove `<ScreenReaderText />`

#3811 - Ensure all ARIA attributes have valid values
- `CarouselFilmStrip.js` - remove erroneous `aria-labelledby` attribute
   - [`aria-labelledby`](https://www.w3.org/TR/WCAG20-TECHS/ARIA7.html) is intended for visible text attributed as the label
   - When this is removed, no new a11y violations are detected
   -  Remove (now)unused ariaID

-  [x] I have added tests and executed them locally
-  [x] I have updated `CHANGELOG.md`
-  [ ] ~I have updated documentation~

## Review Checklist

> This section is for contributors to review your work.

-  [ ] Accessibility reviewed (tab order, content readability, alt text, color contrast)
-  [ ] Browser and platform compatibilities reviewed
-  [ ] CSS styles reviewed (minimal rules, no `z-index`)
-  [ ] Documents reviewed (docs, samples, live demo)
-  [ ] Internationalization reviewed (strings, unit formatting)
-  [ ] `package.json` and `package-lock.json` reviewed
-  [ ] Security reviewed (no data URIs, check for nonce leak)
-  [ ] Tests reviewed (coverage, legitimacy)

### CODE REVIEW

File: __tests__/html/accessibility.activity.stackedLayoutRole.html [hunk 1]
```diff
@@ -14,7 +14,6 @@
         expect,
         host,
         pageObjects,
-        shareObservable,
```
Comment (corinagum) [inline __tests__/html/accessibility.activity.stackedLayoutRole.html:17]: unused in this sample

File: samples/05.custom-components/l.disable-adaptive-cards/index.html [hunk 2]
```diff
@@ -83,7 +83,6 @@
             VideoCardContent
           },
           createStore,
-          createStyleSet,
```
Comment (corinagum) [inline samples/05.custom-components/l.disable-adaptive-cards/index.html:86]: Unused in this sample

File: packages/component/src/Activity/CarouselFilmStrip.js [hunk 3]
```diff
@@ -171,7 +171,6 @@ const CarouselFilmStrip = ({
 
   return (
     <div
-      aria-labelledby={ariaLabelId}
```
Comment (compulim) [inline packages/component/src/Activity/CarouselFilmStrip.js:174]: Should we also remove all references to `ariaLabelId`? On line 130 and 203.

**Review by compulim — APPROVED** [review]
Simpler code. :)

File: packages/component/src/Activity/CarouselFilmStrip.js (hunk 3 above)
Comment (corinagum) [reply packages/component/src/Activity/CarouselFilmStrip.js:174]: Updated, thanks

**Pull request merged**